services/frontend/main.go

```diff
@@ -57,19 +62,42 @@ func run() error {
 		return err
 	}
 
+	autoscalerChan := make(chan event.GenericEvent)
```

Comment:
Are we closing it anywhere? If not, maybe we should as a part of a graceful shutdown.

Reply:
Added 👍